HomeMic/Sources/HomeMic/Home/HomeViewModel.swift

```diff
@@ -3,6 +3,7 @@ import Foundation
 import Navigation
 import DataModels
 
+@MainActor
```

Comment:
this entire class should not be a `@MainActor` since not everything that it does needs to be on the main thread.

`mockTopMovies` and `mockRecentMovies` are already being updated on the mainThread since that is happening inside the `Task`